flask/cli.py

```diff
@@ -41,10 +41,17 @@ def find_best_app(module):
 
     if len(matches) == 1:
         return matches[0]
+
+    # Finally, look for a factory function
+    factory = getattr(module, 'create_app', None)
```

Comment:
I wasn't aware of ``make_app`` in the wild. I've chosen ``create_app`` because it seems to be the convention by virtue of being the example from the docs http://flask.pocoo.org/docs/0.10/patterns/appfactories/. What's the use case for ``make_app``?